### DISCUSSION TITLE
Ability to share a beatmap with a friend on Osu!

### DISCUSSION THREAD

**Discussion by MoutonBinoclard** [ppy/osu#29278] [Ideas] [answered]
I would be great, if instead of sharing link to beatmap in message, you could just right click a beatmap and click share.
Like this, the receiver of the message would not see a link and random number but a nice thumbnail and a dl button
![image](https://github.com/user-attachments/assets/0ffb6cf5-bd4f-49bd-8f2c-5b4472fe3cc6)

**Comment by vatei** [comment, marked as answer]
You can send /np in a chat and it will send the song you're currently listening to. The person will be able to click the link and will be sent to the beatmap page ingame directly!

![image](https://github.com/user-attachments/assets/4ad86dc0-05ab-479e-aeb5-d3ab59ca2846)
![image](https://github.com/user-attachments/assets/80c27ab9-53d9-4145-9219-c36c284547e2)

**Reply by MoutonBinoclard** [reply]
Oh that's nice ! I didn't know that, thanks !
